include/radix/core/state/GameStateManager.hpp

```diff
@@ -1,12 +1,12 @@
-#ifndef RADIX_GAMESTATE_HPP
-#define RADIX_GAMESTATE_HPP
+#ifndef RADIX_GAMESTATE_MANAGER_HPP
+#define RADIX_GAMESTATE_MANAGER_HPP
```

Comment:
Header guard should be RADIX_GAME_STATE_MANAGER_HPP for consistency

Reply:
Good find. I'll fix it in a jiffy.